content/_data/changelogs/weekly.yml

```diff
@@ -8542,6 +8542,64 @@
 
   # pull: 4875 (PR title: Add new line at EOF in config.xml)
 
+- version: '2.251'
+  date: 2020-08-05
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4892
+    issue: 63276
+    authors:
+    - fqueiruga
+    message: |-
+      Empty installed plugins table text is readable again (regression in 2.249).
+  - type: major bug
```

Comment:
Is this major? It was a minor annoying ui issue to me

Reply:
When we presented the changes in the plugin manager user interface, multiple users complained that the table was initially empty when previously it had been initially populated.  The "this page intentionally left blank" text was included so that users had feedback that the list is intentionally empty.

The specific text itself was not as much an issue as the user experience when they are transitioning from a previous Jenkins version to the new plugin manager experience.

Reply:
This isn’t about the plugin manager it’s about the views tabs on the Jenkins home page

Reply:
Sorry, I mistakenly associated your  comment with the plugin manager topic.

I think the tabs not wrapping should be considered major because of the [usability problem](https://issues.jenkins-ci.org/browse/JENKINS-63180) it created with different height entries and a horizontal scroll bar that complicated user navigation.